### PR TITLE
Add MySQL as a data sink in the databuilder

### PR DESCRIPTION
**Pull request by crazy-2020**

### CODE REVIEW

File: rfcs/000-support-mysql-as-backend-store.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+## Guide-level Explanation (aka Product Details)
+
+With the new serializer, data loader, publisher, users could invoke them in ETL job of databuilder if they are using MySQL as the backend store.
+
+## UI/UX-level Explanation
+
+N/A
+
+## Reference-level Explanation (aka Technical Details)
```
Comment (feng-tao) [inline rfcs/000-support-mysql-as-backend-store.md:24]: 1. I think the mysql initialize could be done on the metadata side (e.g either during metadata startup, or provide a cli) per discussion.
2. Should we setup a range dependency which will always use rds latest schema version?
3. I wonder whether you have any prototype and see how long it takes to load metadata to mysql for given N records?

Comment (crazy-2020) [reply rfcs/000-support-mysql-as-backend-store.md:24]: 1. yes, the db init will be done by metadata service.
2. We will probably have to specify the version range of rds package for databuilder to grab the latest one.
3. The metadata publish job(not including extraction, csv loader job) will take 8mins to load 200K records into mysql. (Given CPU: 8, Memory: 8GB, publish_transaction_size: 500, local communication)

Comment (feng-tao) [reply rfcs/000-support-mysql-as-backend-store.md:24]: thanks

**Review by feng-tao — APPROVED** [review]
lgtm

**Pull request merged**